db/migrate_cards/20180414102432_move_customized_skin.rb

```diff
@@ -0,0 +1,23 @@
+# -*- encoding : utf-8 -*-
+
+class MoveCustomizedSkin < Card::Migration
```

Comment:
So, do I understand correctly that:
1. most sites don't have a "customized" skin
2. wikirate needs this because it does, and
3. if another site has one, it may struggle with the decko upgrade?

I'm starting to think we label this 0.5 so that we can claim it was a major upgrade if there are problems :-/

Reply:
I wrote the migration that introduced that customizable skin in November 2017. If someone noticed it and used it in that short period then then he will struggle with the upgrade

Reply:
I added a migration for existing customized skins